circleci-provision-scripts/base.sh

```diff
@@ -47,10 +47,25 @@ EOF
     apt-get update -y
 
     # Install base packages
-    apt-get install \
-        build-essential cmake git-core mercurial zip gdb \
-        htop emacs vim nano lsof vnc4server tmux lzop \
-        build-essential htop emacs vim nano lsof tmux zip vnc4server \
-        curl unzip git-core ack-grep software-properties-common build-essential \
-        parallel libav-tools make autoconf liblapack-dev gfortran
+    cat << EOS | xargs apt-get install
+autoconf
+bsdtar
+build-essential
+cmake
+curl
+gfortran
+git-core
+libav-tools
+liblapack-dev
+lsof
+lzop
+make
+mercurial
+parallel
+software-properties-common
+tmux
+unzip
+vim
+zip
+EOS
```

Comment:
`gfortran` was needed to build `R` [last time I looked at](https://github.com/circleci/image-maker/pull/298#issuecomment-163943979). I'm not sure how many users actually use `R` but Spotify was using and we need to install anyway. So, let me keep it there.

`liblapack-dev` is required to build `SciPy` which is pretty popular and probably needed for other math library so also want to keep in `base` module.